snapcraft/plugins/kbuild.py

```diff
@@ -114,14 +124,44 @@ def __init__(self, name, options, project):
 
     def do_base_config(self, config_path):
         # if kconfigfile is provided use that
+        # elif kconfigflavour is provided, assemble the ubuntu.flavour config
         # otherwise use defconfig to seed the base config
-        if self.options.kconfigfile is None:
+        if self.options.kconfigfile:
+            shutil.copy(self.options.kconfigfile, config_path)
+        elif self.options.kconfigflavour:
```

Comment:
Judging from the descriptions for the other options, perhaps that's a more general comment on the plugin :stuck_out_tongue: .